src/cbapi/psc/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Exported public API for the Cb PSC API
+
+from __future__ import absolute_import
+
+from .rest_api import CbPSCBaseAPI
+from cbapi.psc.models import Device as DeviceV6
```

Comment:
Hmm. I am hesitant to call this DeviceV6 as we don't want to have to maintain this name along with any other names in the future. Do we run into a conflict calling this Device?

Reply:
We might, but I don't know. I can drop the "as DeviceV6" from that import and see if anything barfs at it.

Reply:
In any event, it'll eventually be called Device when the other Device goes away.

Reply:
Looks like the pipeline is still failing

Reply:
Right. But that is not deactivating for 6months so lets not have to maintain DeviceV6

Reply:
It got taken out in the midst of the flurry of changes to get the checks working right. Which they are now. Thank $DEITY. :D